test-packages/ts-ember-preview-types/types/global.d.ts

```diff
@@ -0,0 +1,6 @@
+// Types for compiled templates
+declare module 'ts-ember-app/templates/*' {
+  import { TemplateFactory } from 'htmlbars-inline-precompile';
```

Comment:
Unrelated to this PR, we should probably update the blueprint to either pull this from `ember-cli-htmlbars` or remove the declaration entirely 🤔